docs/playbook/create_team.md

```diff
@@ -0,0 +1,182 @@
+
+## Postman
+
+In Postman create a new tab in which to set up your http request.
+
+If you are unsure how to do this please check the [Postman docs on sending a request](https://learning.postman.com/docs/getting-started/sending-the-first-request/#sending-a-request).
+
+Ensure you are using a `POST` method, as we are about to create a resource on a collection.
+
+Add the endpoint `URL` that we collected in the last step to the input box next to our POST method and append the route information ("api/v1/teams”) to it, e.g.:
+
+```
+https://thecorrecturl.something.yourregion.amazonaws.com/staging/api/v1/teams
+```
+
+Be sure to prefix the route with a “/“
+
+Next, find the tab that says `Body`
+
+When you do this, several radio button options will appear underneath.
+
+- select `raw` & see a new `Text` option appear
+
+- In this new option, select `JSON`
+
+In the larger input panel below these options we add our JSON object that will contain the new team name and the team context.
+
+```
+{
+  “name”: “Customer Service”,
+  “context”: “A”
+}
+```
+
+Still in Postman,
+
+- select the `Authorization` tab
+
+For ”Type”,
+
+- select `API Key` and a panel should appear with the following properties:
+
+```
+Key
+Value
+Add to
+```
+
+The value for _Key_ should be `“x-api-key”`
+
+The value for _Value_ should be the `API Key` you just copied for the `“api-key-social-care-case-viewer-api-staging”` API in the AWS Staging APIs account
+
+The value for _Add to_ should be `“Header”`
+
+Once all this is set up you should be able to send the new request object, containing the data to create a new team, to the staging endpoint
+with the correct authorisation.
+
+In Postman, press the “Send” button and you should receive a response object with the following data (note: the “id” parameter value will return a new integer value for each new response object)
+
+```
+{
+ “id”: 90,
+ “name”: “Customer Service”,
+ “context”: “A”
+}
+```
+
+### --- STEP 4 ---
+
+#### Checking it worked
+
+Go to the [staging Frontend Application](https://social-care-service-staging.hackney.gov.uk/login) ->
+
+Click on the `"Manage workers"` link & enter a workers email to search for ->
+
+Once found, you should see & press the `“Update worker’s details”` button ->
+
+On the following page you will have the `“Team”` drop down, select this and check that it contains your newly added team for the correct service, e.g., Customer Service in the ASC service.
+
+### --- STEP 5 ---
+
+You’ll probably have to do the same in the “Mosaic Production” AWS account
+
+You'll require access to the `Mosaic-Production` AWS account, select the `Management Console`.
+
+Once this account has loaded, go to `API Gateway`,
+
+- select `“APIs”`
+
+- select the correct API: `“mosaic-prod-social-care-case-viewer-api”`
+
+- select `“stages”`
+
+- select `"mosaic-prod”`
+
+- where is says `“Invoke URL:"`
+  copy the value of the URL
+
+```
+https://thecorrecturl.something.yourregion.amazonaws.com/production
```

Comment:
stage is called `mosaic-prod` instead of `production`

Reply:
Yes I put this as "production" as the whole RL is made up, I didn't want to leak any of the details that were in AWS.
Do you think it's misleading?

Reply:
I think it's fine to leave as is i.e. production. I automatically replaced the dummy url with the real one leaving just the stage

Reply:
Cool no worries, thanks for the feedback.

Reply:
You could also wrap parts that require replacement by users in `<>` so `https://<full url from api gateway>/` or `https://<api gateway id>.execute.<aws region>.amazonaws.com/<environment>`.